specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/OrderCreate.json

```diff
@@ -0,0 +1,113 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "TestRG",
+    "orderName": "TestOrderName1",
+    "api-version": "2020-12-01-preview",
+    "User-Agent": [
+      "vscode-restclient"
+    ],
+    "Referer": [
+      "https://localhost/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/TestRG/providers/Microsoft.EdgeOrder/orders/TestOrderName1?api-version=2020-12-01-preview"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "829"
+    ],
+    "orderResource": {
+      "properties": {
+        "orderDetails": {
+          "OrderType": "Purchase",
+          "expectedDataSizeInTeraBytes": 0,
+          "ProductDetails": {
+            "ProductId": "Andrew Cubone"
+          },
+          "Preferences": {
+            "TransportPreferences": {
+              "PreferredShipmentType": "MicrosoftManaged"
+            }
+          }
+        },
+        "addressDetails": {
+          "ShippingAddress": {
+            "Properties": {
+              "ContactDetails": {
+                "ContactName": "first",
+                "Phone": "11111"
+              }
+            },
+            "location": "westus"
+          }
+        }
+      },
+      "Tags": {
+        "mango": "fruit",
+        "carrot": "vegetable"
+      },
+      "location": "westus"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "orderDetails": {
+            "productDetails": {
+              "productId": "Andrew Cubone",
+              "count": 0,
+              "deviceDetails": []
+            },
+            "orderType": "Purchase",
+            "currentStatus": {
+              "orderStatus": "",
+              "lastUpdatedTime": "0001-01-01T05:30:00+05:30"
+            },
+            "orderStatusHistory": [],
+            "preferences": {
+              "transportPreferences": {
+                "preferredShipmentType": "MicrosoftManaged"
+              }
+            },
+            "notificationEmailList": [],
+            "cancellationReason": "",
+            "cancellationStatus": "Cancellable",
+            "deletionStatus": "Allowed",
+            "returnReason": "",
+            "returnStatus": "Returnable"
+          },
+          "addressDetails": {
+            "shippingAddress": {
+              "properties": {
+                "contactDetails": {
+                  "contactName": "first",
+                  "phone": "11111"
+                }
+              },
+              "location": "westus",
+              "tags": {},
+              "type": "Microsoft.EdgeOrder/addresses"
+            }
+          },
+          "startTime": "0001-01-01T05:30:00+05:30",
+          "orderCollectionId": ""
```

Comment:
>orderCollectionId [](start = 11, length = 17)

this is defined as a readonly property - how does the service set this value in the response? what is the sequence in which orders and orderCollections need to be created?

Reply:
ok, I'm thinking this is empty/null when the order is created - but when the ordercollection is created this will be populated

---
In reply to: [547566441](https://github.com/Azure/azure-rest-api-specs/pull/12142#discussion_r547566441) [](ancestors = 547566441)

Reply:
Yes while creating order collection. We will update order object with order collection Id